resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackend.scala

```diff
@@ -324,11 +324,24 @@ private[spark] class KubernetesClusterSchedulerBackend(
           super.receiveAndReply(context)))
 
     override def onDisconnected(rpcAddress: RpcAddress): Unit = {
```

Comment:
So this is tested by "Test decommissioning with dynamic allocation & shuffle cleanups", right?

But that test is not even a decommissioning test:
https://github.com/apache/spark/blob/master/resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DecommissionSuite.scala#L183

I am a bit confused but I try to understand where and how this is tested. 
Could you please give me some pointers?

Reply:
So the decommissionsuite will end up running through the "onDisconnect" once the executor exits.

Reply:
Sorry I completely forget about this. Now I get how it get it how it will be called.